user-guide/modules/ROOT/pages/getting-started-with-macos.adoc

```diff
@@ -1,6 +1,45 @@
 = Getting Started with macOS
 
-To install Boost libraries on macOS, you can use Homebrew, a popular package manager for macOS that simplifies the installation of software packages. Here's a step-by-step guide to installing the Boost libraries using Homebrew.
+To install and build Boost libraries on macOS, you can either use CMake, or install and use a package manager such as Homebrew, a popular package manager for macOS that simplifies the installation of software packages. 
+
+== Install Boost using CMake
+
+If you do not wish to use an package manager, installing Boost libraries on macOS using CMake involves several steps. Here's an outline of the process:
+
+. Open a terminal window and navigate to the directory where you extracted the Boost library files.
+
+. Run the bootstrap.sh script to generate the build system files for your platform. For example, to build Boost with clang on macOS, run:
+
+[source,txt]
+----
+./bootstrap.sh --with-toolset=clang
```

Comment:
Shouldn't this be just `toolset=clang` instead of `--with-toolset=clang`

Reply:
OK - corrected